site/soc/software/config/endpoints.yaml

```diff
@@ -76,18 +76,16 @@ data:
         default: null
         public:
           host: keystone.DOMAIN
-        admin:
-          host: keystone.DOMAIN
       path:
         default: /v3
       scheme:
-        # default: "https"
         default: "http"
+        public: "{{ ucp_endpoint_http_scheme }}"
         internal: "http"
       port:
         api:
-          # default: 443
           default: 80
```

Comment:
if you choose not to expose admin end points, why don't we leave admin port to 5000 as well?

Reply:
yes, admin endpoint needs to be same as internal endpoint as its primarily used for token validation by services. Will change it to map to internal endpoint.